test/emu_mps/test_hamiltonian.py

```diff
@@ -324,6 +329,50 @@ def test_5_qubit(hamiltonian_type):
     )
 
 
+@pytest.mark.parametrize(
+    "hamiltonian_type",
+    [
+        HamiltonianType.Rydberg,
+        HamiltonianType.XY,
+    ],
+)
+@pytest.mark.parametrize(
+    "system_size",
+    [10, 25, 50],
+)
+def test_N_qubit(hamiltonian_type, system_size):
```

Comment:
this test should also move to the emu_base config

Reply:
done